runtime-light/crypto/crypto-builtins.h

```diff
@@ -0,0 +1,10 @@
+//  Compiler for PHP (aka KPHP)
+//  Copyright (c) 2024 LLC «V Kontakte»
+//  Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include "runtime-core/runtime-core.h"
+#include "runtime-light/coroutine/task.h"
+
+task_t<Optional<string>> f$openssl_random_pseudo_bytes(int64_t length);
```

Comment:
Let's mark it `noexcept`.

Reply:
`string::append` is not marked as `noexcept`, so I don't think I should

Reply:
We will be compiling with `-fno-exception` in the future. Basically `noexcept` for us is just a note to the compiler that it can generate better code with respect to our decision not to care about exceptions.

Reply:
I think it's strange to mark function that call no-`noexcept` functions as `noexcept`, may be we should check all the functions once in another PR and paste `noexcept`? Most of other runtime functions that are dealing with streams are not `noexcept`, too.

Reply:
They are now actually :) Also, `noexcept` does nothing except it tells the compiler it's required to generate specific code that helps in exception handling.